config/routes.rb

```diff
@@ -27,12 +27,14 @@
   resources :contact_methods
   get "/combined_form", to: "contributions#combined_form", as: "combined_form"
   get "/thank_you", to: "contributions#thank_you", as: "contribution_thank_you"
-  resources :contributions, only: [:index] do
+  resources :contributions, only: [:index, :show] do
     member do
       get "/respond", to: "contributions#respond", as: "respond"
       get "/triage", to: "contributions#triage", as: "triage"
       patch "/triage", to: "contributions#triage_update"
       post "/triage", to: "contributions#triage_update"
+      post :claim_contribution
+      get :claim_contribution_form
```

Comment:
I think this could be better modeled as a nested resource pointing at a new controller:
```
resources :contributions, only: [:index, :show] do
  member do
    # snipped ...
  end
  
  resources :claims, only: [:new, :create]
```
`claim_contribution_form` would then map to the `new` action and `claim_contribution` to `create`.

I realize we had already gone down the non-resourceful route (pun intended! :) with `/triage` and `/respond`, so quite understandable that y'all followed that pattern. OTOH, i do think the resourceful version is a better separation of responsibilities and helps keep the `ContributionsController` small and coherent. It also sets us up better if in the future we need to, eg, revoke a claim (which would map nicely to the destroy action).

If we don't have time to refactor in this direction, that's definitely fine!

Reply:
I like this idea. I have moved all the logic to `ClaimsController` now.